dbt/models/default/schema/default.vw_pin_condo_char.yml

```diff
@@ -89,6 +89,20 @@ models:
             - year
           config:
             error_if: ">70297"
+      - not_null:
+          name: default_vw_pin_condo_char_card_not_null
+          column_name: card
```

Comment:
**suggestion:** For simple tests like these, Jean suggested it's probably better to attach them to their respective columns rather than defining them as standalone tests (we get more metadata that way). That said, we aren't exporting them in any meaningful way, so I'm not sure it matters. Cue a longer conversation about unit vs data tests (or exported vs CI tests).

My original goal of defining tests on the default views was to test the _underlying data_, however it seems like almost all of that function has moved to the `test_qc_` tests. Perhaps we should only put data QC tests on sources (iasworld, other ingested stuff) and use other tests as CI checks only. Any thoughts @jeancochrane @wrridgeway?

Reply:
Personally, I'm for it. I just had to remove the `card_proration_rate` test for null-ness since the fields it's built on are absolute garbage. Having a view fail every time we try to build it because oby and comdat are poorly maintained doesn't seem like the proper way to address issues in iasworld.

Reply:
Agreed @dfsnow. Once unit tests are released in dbt 1.8 (current target date is 5/9) and we can get started on https://github.com/ccao-data/data-architecture/issues/382 it'll be a lot easier for us to differentiate these two types of tests.